### DISCUSSION TITLE
Superset helm update to 3.1

### DISCUSSION THREAD

**Discussion by tanmaykansara** [apache/superset#26676] [General]
Hi
Any thoughts on when helm chart would be updated to install Superset 3.1 that is now released?
Thanks

**Comment by michael-s-molina** [comment]
@craig-rueda

**Comment by craig-rueda** [comment]
Thanks for raising this!
It should work as-is, AFAIK just by simply picking the `image.tag` you're interested for the time being. We will make a PR to update the default values in the coming days.

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/pull/26707
